.github/workflows/review.yml

```diff
@@ -43,3 +43,26 @@ jobs:
       run: mv .github/tools/.checkpatch.conf .
     - name: Run checkpatch review
       uses: webispy/checkpatch-action@v9
+  upstream_tests:
+     runs-on: self-hosted
+     timeout-minutes: 120
```

Comment:
120 minutes?  Is that accurate?  What is the average test time?

Reply:
I changed it to the lower value - 90 minutes, and unfortunately tests on this machine takes more than 1 hour. Discussed with Mariusz and he is OK with waiting for the results. 
It depends on platform, the fastest run which I saw was 30 minutes.